core/src/main/kotlin/entity/application/ApplicationCommand.kt

```diff
@@ -45,16 +47,27 @@ public sealed interface ApplicationCommand : ApplicationCommandBehavior {
      */
     public val version: Snowflake get() = data.version
 
+    /**
+     * Set of [Permissions] required to execute this command unless a server admin changed them.
+     */
+    public val defaultMemberPermissions: Permissions? get() = data.defaultMemberPermissions.value
+
     /**
      * whether the command is enabled by default when the app is added to a guild.
      */
+    @Deprecated("'defaultPermission' is deprecated in favor of 'defaultMemberPermissions' and 'dmPermission'.")
     public val defaultPermission: Boolean? get() = data.defaultPermission.discordBoolean
 
 
 }
 
 
-public interface GlobalApplicationCommand : ApplicationCommand, GlobalApplicationCommandBehavior
+public interface GlobalApplicationCommand : ApplicationCommand, GlobalApplicationCommandBehavior {
+    /**
+     * Whether this command can be executed in dms or not
+     */
+    public val dmPermission: Boolean? get() = data.dmPermission.value
```

Comment:
```suggestion
    /**
     * Whether this command is available in DMs with the application.
     */
    public val dmPermission: Boolean get() = data.dmPermission.orElse(true)
```
commands are usable in DMs by default

Reply:
Isn't damper mission null by default?

Reply:
Sure, but `null`/missing means it's available in DMs, so why expose null here in core? (if only chat input commands can be used in DMs, this should also be moved to the global chat input command entity)

Reply:
I don't think dmÜermission should show, whether it's available in DMs, it should show the state of the permission, and that can be null

Reply:
Hm ok, probably not very important anyway.

Reply:
The reason for this suggestion was the description of the `dm_permission` field:
> Indicates whether the command is available in DMs with the app, only for globally-scoped commands. By default, commands are visible.